File: client/src/app/model/PM/StaffWeeklyPlanDto.ts
export interface StaffWeeklyPlanDto {
    activityNo: string;
    activity: string;
    placeOfWork: string;
    executionDate: Date;
    responsible: string;
}

export interface FilterDateCriteriaDto {
    fromDate: Date;
    toDate: Date;
}


export interface PlanPerformanceListDto {
    activityNo: string;
    activity: string;
    status: string;
    reasonsForNotComplited: string;
    remark: string;
}

File: IntegratedDigitalAPI/IntegratedImplementation/DTOS/PM/StaffWeeklyPlanDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IntegratedImplementation.DTOS.PM
{
    public class StaffWeeklyPlanDto
    {
        public string ActivityNo { get; set; } = null!;
        public string Activity { get; set; } = null!;
        public string PlaceOfWork { get; set; } = null!;
        public DateTime ExecutionDate { get; set; }
        public string Responsible { get; set; } = null!;
    }

    public class FilterDateCriteriaDto
    {
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
    }


    public class PlanPerformanceListDto
    {
        public string ActivityNo { get; set; } = null!;
        public string Activity { get; set; } = null!;
        public string Status { get; set; } = null!;
        public string ReasonsForNotComplited { get; set; } = null!;
        public string Remark { get; set; } = null!;
    }

}
